certified-connectors/Converter/apiDefinition.swagger.json

```diff
@@ -0,0 +1,204 @@
+        "consumes": [
+          "text/plain"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-visibility": "important",
+        "summary": "Convert CSV To JSON",
+        "description": "Convert CSV-Data to JSON",
+        "parameters": [
+          {
+            "name": "CSV-Data",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-ms-visibility": "important",
+              "title": ""
+            },
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/ConvertJsonToCsvdata": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "summary": "Convert JSON To CSV",
+        "operationId": "ConvertJsonToCsvdata",
+        "parameters": [
+          {
+            "name": "JSON",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-ms-visibility": "important"
+            },
+            "x-ms-visibility": "important"
+          }
+        ],
+        "description": "Convert JSON-Data to CSV-Data",
+        "x-ms-visibility": "important"
+      }
+    },
+    "/ConvertSharePointSearchResults": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "title": {
+                    "type": "string",
+                    "description": "Title of SharePoint-Ressource",
+                    "title": "title"
+                  },
+                  "link": {
+                    "type": "string",
+                    "description": "Link of SharePoint-Ressource",
+                    "title": "link"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Convert SharePoint Search Results",
+        "description": "Convert SharePoint search results to a simple list",
+        "operationId": "ConvertSharePointSearchResults",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "important",
+            "description": "SharePoint Json-Result",
+            "x-ms-summary": "SP-Result"
+          },
+          {
+            "name": "JSON",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-ms-visibility": "important",
+              "title": ""
```

Comment:
@stsiek Empty property values can be removed until unless you have a specific reason to keep them.